lib/tasks/device_story_tasks.rake

```diff
@@ -3,6 +3,6 @@
 namespace :device_story do
   desc 'Update device metadata from ttserve'
   task update_metadata: :environment do
-    Tasks::DeviceStory::UpdateMetadata.call
+    Tasks::DeviceStories::UpdateMetadata.call
```

Comment:
I renamed this task from `Tasks::DeviceStory::UpdateMetadata` to `Tasks::DeviceStories::UpdateMetadata` to avoid class naming conflicts with `DeviceStory` Active record